lib/fs.js

```diff
@@ -537,8 +561,13 @@ fs.open = function(path, flags, mode, callback_) {
   var req = new FSReqWrap();
   req.oncomplete = callback;
 
+  var intFlags = stringToFlags(flags);
+  if (intFlags === undefined) {
```

Comment:
To be honest, I'm not sure if it could ever be zero... I could also do a `Number.isInteger` test, but I figured this would execute faster.